pkg/kapp/resources/file_resources.go

```diff
@@ -32,7 +32,7 @@ func NewFileResources(file string) ([]FileResource, error) {
 	default:
 		fileInfo, err := os.Stat(file)
 		if err != nil {
-			return nil, fmt.Errorf("Checking file '%s'", file)
+			return nil, fmt.Errorf("Checking file '%s': %v", file, err)
```

Comment:
Maybe we should remove the filename from the error message now as the error returned from os.Stat always includes the file name?

Reply:
good point,  [the docs](https://pkg.go.dev/os) say very clearly that stat will include the filename in the error.  I was then tempted to get rid of the 'Checking file' string but i think that it provides a sort of SEO / reverse-error engineering, by giving someone something to grep for in the code.